nixos/modules/services/continuous-integration/gitlab-runner.nix

```diff
@@ -15,6 +15,23 @@ in
       description = "Verbatim config.toml to use";
     };
 
+    graceful = mkOption {
```

Comment:
I tought about `gracefulTermination` as well as this most accurately describes what it actually does.